flow/flow.h

```diff
@@ -687,6 +687,11 @@ class Future
 	{
 		sav->send(presentValue);
 	}
+	Future(T&& presentValue)
+		: sav(new SAV<T>(1, 0))
+	{
+		sav->send(std::forward<T>(presentValue));
```

Comment:
Nice find!

This is an rvalue reference, not a [forwarding reference](https://en.cppreference.com/w/cpp/language/reference#Forwarding_references), so I think std::move would be clearer

Reply:
Ah right, I actually only tried forward as a last resort because I wasn't seeing the test output I expected with move at first but it was a different issue.  Fixed.